python/shotgun_model/shotgunmodel.py

```diff
@@ -948,6 +958,12 @@ def __rebuild_whole_tree_from_sg_data(self, data):
         root = self.invisibleRootItem()
         self.__populate_complete_tree_r(data, root, self.__hierarchy, {})
 
+        # Turn our signals back on as we have effectively reset the model at
+        # this point and finally call the QStandardItemModel's endResetModel
+        # method.
+        self.blockSignals(False)
```

Comment:
I think that should work!  I don't recall why I had put this here, perhaps because a signal existed to notify when the tree was repopulated?  Honestly I don't recall.